### PR TITLE
[FIX] web_editor: fix unrecognized image upload

### PR DESCRIPTION
**Pull request by rdeodoo**
Since 3765ac1f1f16, uploading an image not recognized as such by PIL would not
behaves as it should.

Step to reproduce:
  - Enter edit mode, drag & drop a snippet with an image
  - Double click on the image to open media dialog
  - Upload a local image of type `.webp`
  - 2 toaster are shown:
    1. A progress bar toaster with a generic message "File could not be saved"
    2. A generic warning toaster on top of the previous one with the correct
       error message "[..] format is not supported. Try with: .gif [..]"

Instead, only the progress bar toaster should be shown, and the error message
should be displayed in it.

task-2607393

### CODE REVIEW

**Comment by rdeodoo** [comment]
2 lines fix, but comes with a complete testing suite
Hopefully the feature should not break again..

File: addons/web_editor/controllers/main.py [hunk 1]
```diff
@@ -173,6 +173,10 @@ def add_data(self, name, data, is_image, quality=0, width=0, height=0, res_id=Fa
                 mimetype = guess_mimetype(b64decode(data))
                 if mimetype not in SUPPORTED_IMAGE_MIMETYPES:
                     return {'error': format_error_msg}
+            except UserError:
+                # considered as an image by the brower file input, but not
```
Comment (JKE-be) [inline addons/web_editor/controllers/main.py:177]: browser

Comment (rdeodoo) [reply addons/web_editor/controllers/main.py]: Done

**Comment by JKE-be** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**